### PR TITLE
Texture mipmaps serialization

### PR DESCRIPTION
**Pull request by takahirox**
Separating Texture mipmaps serialization from #13915

See https://github.com/mrdoob/three.js/pull/13915#discussion_r183960083

### CODE REVIEW

File: src/loaders/ObjectLoader.js [hunk 1]
```diff
@@ -569,6 +569,43 @@ Object.assign( ObjectLoader.prototype, {
...
+						Uint8Array: Uint8Array,
+						// Workaround for IE11 pre KB2929437. See #11440
+						Uint8ClampedArray: typeof Uint8ClampedArray !== 'undefined' ? Uint8ClampedArray : Uint8Array,
+						Int16Array: Int16Array,
+						Uint16Array: Uint16Array,
+						Int32Array: Int32Array,
+						Uint32Array: Uint32Array,
+						Float32Array: Float32Array,
+						Float64Array: Float64Array
+					};
```
Comment (takahirox) [inline src/loaders/ObjectLoader.js]: Wondering if we can remove this table...

File: src/textures/Texture.js [hunk 2]
```diff
@@ -170,6 +170,29 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 		}
 
+		function addImage( image ) {
+
+			// TODO: Move to THREE.Image
+
+			if ( image.uuid === undefined ) {
+
+				image.uuid = _Math.generateUUID(); // UGH
```
Comment (takahirox) [inline src/textures/Texture.js]: Off topic: I wanna remove this pollution.

**Comment by Mugen87** [comment]
Let's revisit this issue when #17745 goes in.

File: src/textures/Texture.js (hunk 2 above)
Comment (Mugen87) [reply src/textures/Texture.js]: This will be solved via #17949.

**Pull request closed without merging**